pyproject.toml

```diff
@@ -52,7 +52,7 @@ requires = [
 
 [tool.cibuildwheel]
 # See: https://cibuildwheel.readthedocs.io/en/stable/options/#configuration-file
-build-verbosity = "3"
+build-verbosity = 3
```

Comment:
I believe `build-verbosity` is supposed to be an integer, not a string: https://cibuildwheel.pypa.io/en/stable/options/#configuration-file:~:text=pass%20%3D%20%5B%5D%0Abuild%2D-,verbosity,-%3D%200%0A%0Abefore